cmd/list.go

```diff
@@ -76,3 +94,23 @@ func key(s string) string {
 	secretKey := tokens[1]
 	return secretKey
 }
+
+type ByName []store.Secret
```

Comment:
This seems unused, we should remove it if that's the case.

Reply:
It's required by the 3 lines which follow it.

Reply:
ah, I missed line 58, didn't realize that you always sort by name first.